### DISCUSSION TITLE
Magit v3.1.0 released

### DISCUSSION THREAD

**Discussion by tarsius** [magit/magit#4425] [Announcements]
Another one already!

* [Announcement](https://emacsair.me/2021/07/01/magit-3.1/)
* [Release Notes](https://github.com/magit/magit/blob/master/Documentation/RelNotes/3.1.0.org)

<hr /><em>This discussion was created from the release <a href='https://github.com/magit/magit/releases/tag/v3.1.0'>v3.1.0</a>.</em>

**Comment by Ostars-eye** [comment]
é magica =)
